src/Quarrel/App.xaml.cs

```diff
@@ -138,32 +198,68 @@ public void SetupTitleBar()
                 StatusBar statusBar = StatusBar.GetForCurrentView();
                 if (statusBar != null)
                 {
-                    statusBar.BackgroundOpacity = 1;
-                    statusBar.BackgroundColor = ((SolidColorBrush)Current.Resources["AcrylicCommandBarBackground"]).Color;
-                    statusBar.ForegroundColor = ((SolidColorBrush)Current.Resources["MessageForeground"]).Color;
+                    try
+                    {
+                        statusBar.BackgroundOpacity = 1;
+                        statusBar.BackgroundColor = ((SolidColorBrush)Current.Resources["AcrylicCommandBarBackground"]).Color;
+                        statusBar.ForegroundColor = ((SolidColorBrush)Current.Resources["MessageForeground"]).Color;
+                    }
+                    catch (Exception ex) {
+                        Logger.LogError(new EventId(), ex, "Error caught accessing resources. (Group 1)");
+                        _lightOnly = true;
+                    }
```

Comment:
Won't this disable changing theme on mobile as this will always fail on mobile due to lack of acrylic support?

Reply:
No, line 209 actually won't affect anything as the check has already happened.  Lines 209 and 227 should be removed.